### PR TITLE
Issue/271 dash

### PR DESCRIPTION
**Pull request by iperdomo**

### CODE REVIEW

File: docker/dash/backend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:14.04
+MAINTAINER Akvo Foundation <devops@akvo.org>
+
+# Configure locales
```
Comment (orifito) [inline docker/dash/backend/Dockerfile]: do we need to configure locales?

File: docker/dash/backend/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+RUN lein
+
+WORKDIR /opt/dash
+
+RUN git clone https://github.com/akvo/akvo-dash.git && \
+    cd akvo-dash/backend && \
+    lein deps && lein uberjar && \
+    rm -rf /opt/dash/.m2
+
+EXPOSE 3000
```
Comment (orifito) [inline docker/dash/backend/Dockerfile]: I prefer to specify exposed ports into docker-compose.yml

File: docker/dash/backend/Dockerfile (hunk 1 above)
Comment (iperdomo) [inline docker/dash/backend/Dockerfile]: we need to make sure that LANG environment is `en_US.UTF8`, we have had issues in the past and we have had to force it e.g. https://github.com/akvo/akvo-flow-services/commit/66cda7452798c8b96d6b8584ef354c8937ee0379

File: docker/dash/backend/Dockerfile (hunk 2 above)
Comment (iperdomo) [inline docker/dash/backend/Dockerfile]: Perfect, i'll remove the expose

File: docker/dash/backend/Dockerfile (hunk 1 above)
Comment (orifito) [inline docker/dash/backend/Dockerfile]: ah! :cool:

**Pull request merged**